library/src/main/java/com/novoda/downloadmanager/DownloadErrorFactory.java

```diff
@@ -41,8 +41,9 @@ static DownloadError createCannotWriteToFileError(DownloadFileStatus downloadFil
         return new DownloadError(DownloadError.Type.FILE_CANNOT_BE_WRITTEN, cannotWriteToFileMessage);
     }
 
-    static DownloadError createStorageNotAvailableError(FilePersistence filePersistence) {
-        String storageUnavailableMessage = "Storage with base path: " + filePersistence.basePath() + " is not available";
+    static DownloadError createStorageNotAvailableError(FilePath filePath) {
+        // QUESTION: is it enough to swap out the basePath with the filePath?
```

Comment:
yeah I guess, can we test this?

Reply:
Ah! I just realised that this isn't actually exposed in the `download-manager` anymore 🤔 We are using absolute paths and not checking if the storage is mounted so we'll see an error when attempting to write instead. Is it ok to remove this @zegnus ?

Reply:
ok 👍